group_name: "manipulator"
base_frame: "world"
tool_frame: "flange"
poi_names: ["a_forearm","b_mount_flange","b_forearm","flange"]

max_cart_acc: 2.5
Tr: 0.15
min_distance: 0.2
v_h: 1.6

n_objects: 1
min_range: [-20,-20, 0]
max_range: [ 20, 20, 20]
norm_1: true

n_iter: 500000
database_name: "ssm_dataset_scaling_simple"
